templates/public-adventures.html

```diff
@@ -0,0 +1,62 @@
+
+{% extends "auth.html" %}
+
+{% block regular_content %}
+
+    <div id="filters"
+        class="flex flex-col sm:flex-row items-center gap-2
+        justify-around bg-blue-400 shadow-md rounded-lg p-4 mb-4"
+        _="on load set $filtered to []">
+        <div>{{_('apply_filters')}}</div>
+        <div id="search" class="flex-1">
+            <input id="search_adventure"
+                type="search"
+                placeholder="{{_('search')}}"
+                class="border text-black border-gray-500 rounded h-8 px-2 w-full"
+                _="on keyup debounced at 500ms set searchTerm to my value 
+                    hedyApp.applyFilter(searchTerm, 'search', $filtered)">
```

Comment:
Can we apply the same UI in this filter as the one we use to filter programs in the /explore and /programs page? We are trying to be a bit more consistent with our UX.